opentelemetry-sdk/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## vNext
 
+- **Feature**: Added Rust code generation for profiles protos. [#2979](https://github.com/open-telemetry/opentelemetry-rust/pull/2979)
```

Comment:
Can you place it in opentelemetry-proto crate's changelog.

Reply:
done thanks for catching